net/aircrack-ng/patches/0001-osdep-ratiotap-fix-compilation-on-Linux.patch

```diff
@@ -0,0 +1,36 @@
+From 85fde8b32b2095d60cead683c958487701221125 Mon Sep 17 00:00:00 2001
+From: Yousong Zhou <yszhou4tech@gmail.com>
+Date: Wed, 30 Dec 2015 10:39:56 +0800
+Subject: [PATCH] osdep: ratiotap: fix compilation on Linux
+
+le16toh and le32toh should be already available by platform.h
```

Comment:
I can't allow a non-upstreamed patch into openwrt, and this patch is too over my head to accept upstream.  Can you please open a PR for aircrack-ng including the build error in the comments?  If this fix it right we should get it  out to everyone.